shared/message/platform/NUSenseData.proto

```diff
@@ -31,38 +31,40 @@ syntax = "proto3";
 
 package message.platform;
 
-// Message mainly for NUSense <-> NUC communication. NUSense will encode this message via nanopb after it queries all
-// servo states and send it to the NUC.
-message Servo {
-    /// The ID of the servo
-    uint32 id = 1;
-    /// Current error state of the servo
-    /// different to the dynamixel packet error status
-    uint32 hardware_error = 2;
-    /// Whether the servo's torque is enabled, allowing it to move
-    bool torque_enabled = 3;
-    // Values that are either simulated or read from the servos
-    /// The last read pulse width modulation of the servo
-    float present_pwm = 4;
-    /// The last read current of the servo
-    float present_current = 5;
-    /// The last read velocity of the servo
-    float present_velocity = 6;
-    /// The last read position of the servo
-    float present_position = 7;
-    // The desired pwm value for motor control
-    float goal_pwm = 8;
-    // The desired current value for the servo
-    float goal_current = 9;
-    // The desired velocity value for the servo
-    float goal_velocity = 10;
-    // The desired position value for the servo
-    float goal_position = 11;
-    /// The last read voltage of the servo
-    float voltage = 12;
-    /// The last read temperature of the servo
-    float temperature = 13;
-}
+import "message/actuation/Servos.proto";
+
+// // Message mainly for NUSense <-> NUC communication. NUSense will encode this message via nanopb after it queries all
+// // servo states and send it to the NUC.
+// message Servo {
+//     /// The ID of the servo
+//     uint32 id = 1;
+//     /// Current error state of the servo
+//     /// different to the dynamixel packet error status
+//     uint32 hardware_error = 2;
+//     /// Whether the servo's torque is enabled, allowing it to move
+//     bool torque_enabled = 3;
+//     // Values that are either simulated or read from the servos
+//     /// The last read pulse width modulation of the servo
+//     float present_pwm = 4;
+//     /// The last read current of the servo
+//     float present_current = 5;
+//     /// The last read velocity of the servo
+//     float present_velocity = 6;
+//     /// The last read position of the servo
+//     float present_position = 7;
+//     // The desired pwm value for motor control
+//     float goal_pwm = 8;
+//     // The desired current value for the servo
+//     float goal_current = 9;
+//     // The desired velocity value for the servo
+//     float goal_velocity = 10;
+//     // The desired position value for the servo
+//     float goal_position = 11;
+//     /// The last read voltage of the servo
+//     float voltage = 12;
+//     /// The last read temperature of the servo
+//     float temperature = 13;
+// }
```

Comment:
```suggestion

```

I agree with removing this message as this allows us to just fill up a ServoState message in NUsense so we can send that to the NUC and it can be emitted straight away into the system instead of suffering from the overhead of having to fill up RawSensors and stuff.

This implies that the message hash in NUController/NUSense will have to be changed and the Servo message will have to be regenerated.

Another note, ServoTargets and SubcontrollerServoTargets uses a google.protobuf.Timestamp and .Duration respectively. We might wanna add that to the Servo message as well.